pkgs/development/python-modules/wavedrom/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, attrdict
+, cairosvg
+, pillow
+, pytestCheckHook
+, setuptools-scm
+, six
+, svgwrite
+, xmldiff
+}:
+
+buildPythonPackage rec {
+  pname = "wavedrom";
+  version = "2.0.3.post2";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "239b3435ff116b09007d5517eed755fc8591891b7271a1cd40db9e400c02448d";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    attrdict
+    svgwrite
+    six
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    xmldiff
+    pillow
+    cairosvg
+  ];
+
+  postPatch = ''
+    sed -i 's|use_scm_version=True|version="${version}"|' setup.py
+  '';
```

Comment:
Please use SETUPTOOLS_SCM_PRETEND_VERSION env instead.

Reply:
Thanks for the hint! Fixed in b0555f0.